library/src/main/java/com/novoda/downloadmanager/lib/BatchRepository.java

```diff
@@ -3,7 +3,7 @@
 import android.content.ContentResolver;
 import android.content.ContentValues;
 import android.database.Cursor;
-import android.util.SparseIntArray;
+import android.support.v4.util.SparseArrayCompat;
```

Comment:
Using the compat version allows us to test the functionality, although does mean we get an autoboxing penalty.